recipes/mayavi/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "4.5.0" %}
+
+package:
+  name: mayavi
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/enthought/mayavi.git
+  git_rev: {{ version }}
+
+build:
+  entry_points:
+    - mayavi2 = mayavi.scripts.mayavi2:main
+    - tvtk_doc = tvtk.tools.tvtk_doc:main
+  skip: True  # [osx or (linux and py34) or win]
```

Comment:
just because the [VTK dependencies is skipped for linux and py34](https://github.com/conda-forge/vtk-feedstock/blob/487d1d33b5f6ce6995454f482928e34520823613/recipe/meta.yaml#L26), I'll add a comment about that